docker/event-store-init.sql

```diff
@@ -5,6 +5,8 @@ create table catalog.events (
     stream_id uuid NOT NULL,
     version bigint NOT NULL,
 
+    correlation_id uuid,
```

Comment:
the correlation id is generated as uuid to make it unique, but it is a generic string
(same for next occurrences)

Reply:
Thanks, fixed!